planner/core/logical_plan_trace_test.go

```diff
@@ -101,6 +101,36 @@ func (s *testPlanSuite) TestSingleRuleTraceStep(c *C) {
 				},
 			},
 		},
+		{
+			sql:            "select count(*) from t a , t b, t c",
+			flags:          []uint64{flagBuildKeyInfo, flagPrunColumns, flagPushDownAgg},
+			assertRuleName: "aggregation_push_down",
+			assertRuleSteps: []assertTraceStep{
+				{
+					assertAction: "aggFuncs[count(Column#38)] pushed into join[5] right path",
```

Comment:
Just comment. Is it better to add the expression here?

Reply:
Currently not necessary.